lib/ansible/modules/apt.py

```diff
@@ -63,44 +63,43 @@
     default: 'no'
   default_release:
     description:
-      - Corresponds to the C(-t) option for I(apt) and sets pin priorities
+      - Corresponds to the C(-t) option for C(apt) and sets pin priorities.
```

Comment:
we are referencing apt as a system, not a CLI tool, this actually predates the apt CLI tool, there was only `apt-get`

Reply:
Got it, will be fixing that later tomight (UTC+12).

Reply:
```suggestion
      - Corresponds to the C(-t) option for I(apt) and sets pin priorities.
```